controllers/cleanup.go

```diff
@@ -88,9 +88,4 @@ func (r *SearchReconciler) deleteLegacyServiceMonitorSetup(instance *searchv1alp
 	if err = r.Delete(r.context, r.MetricsRoleBinding(instance)); err != nil && !errors.IsNotFound(err) {
 		log.Error(err, "Failed to remove RoleBinding")
 	}
```

Comment:
Let's add a simple log that we could use to confirm that this function executed. We can infer that it was successful if we don't see any errors logged before.

```suggestion
	}
	log.Info("Done deleting legacy service monitors.");
```

Reply:
Done